akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/PropsAdapter.scala

```diff
@@ -23,13 +23,13 @@ import akka.dispatch.Mailboxes
 
   private final val TypedCreatorFunctionConsumerClazz = classOf[TypedCreatorFunctionConsumer]
   private final val ActorAdapterClazz = classOf[ActorAdapter[_]]
-  private final val DefaultTypedDeploy = Deploy.local.copy(mailbox = "akka.actor.typed.default-mailbox")
+  private final val DefaultTypedDeploy = Deploy.local
```

Comment:
why is this changed? that mailbox is supposed to be the default
I think this would change the semantics of below
```
case _: DefaultMailboxSelector           => DefaultTypedDeploy
```